tasks/actorstate/miner.go

```diff
@@ -46,48 +72,19 @@ func (m StorageMinerExtractor) Extract(ctx context.Context, a ActorInfo, node Ac
 		return nil, xerrors.Errorf("creating miner state extraction context: %w", err)
 	}
 
-	minerInfoModel, err := ExtractMinerInfo(ctx, a, ec)
-	if err != nil {
-		return nil, xerrors.Errorf("extracting miner info: %w", err)
-	}
-
-	lockedFundsModel, err := ExtractMinerLockedFunds(ctx, a, ec)
-	if err != nil {
-		return nil, xerrors.Errorf("extracting miner locked funds: %w", err)
-	}
-
-	feeDebtModel, err := ExtractMinerFeeDebt(ctx, a, ec)
-	if err != nil {
-		return nil, xerrors.Errorf("extracting miner fee debt: %w", err)
-	}
-
-	currDeadlineModel, err := ExtractMinerCurrentDeadlineInfo(ctx, a, ec)
-	if err != nil {
-		return nil, xerrors.Errorf("extracting miner current deadline info: %w", err)
-	}
-
-	preCommitModel, sectorModel, sectorDealsModel, sectorEventsModel, err := ExtractMinerSectorData(ctx, ec, a, node)
-	if err != nil {
-		return nil, xerrors.Errorf("extracting miner sector changes: %w", err)
-	}
-
-	posts, err := ExtractMinerPoSts(ctx, &a, ec, node)
-	if err != nil {
-		return nil, xerrors.Errorf("extracting miner posts: %v", err)
+	var out model.PersistableList
+	for _, m := range a.Models {
```

Comment:
Can we make this concurrent? So each extraction is in a separate goroutine. Does the diffcache support that?

Reply:
Will add concurrency in a follow-on. Since these tasks never ran concurrently, to begin with, I expect any performance impact resulting from this change to be minimal.